theme/layout/breakpoints.less

```diff
@@ -1,12 +1,18 @@
 // Breakpoint
 @responsive-breakpoint: 740px;
 
+@mobile: %(~"only screen and (max-width: %s)", (@responsive-breakpoint - 1));
+
+@desktop: %(~"only screen and (min-width: %s)", @responsive-breakpoint);
+
+// DEPRECATED: Use @mobile variable instead.
 .smallDeviceOnly(@rules) {
   @media only screen and (max-width: (@responsive-breakpoint - 1)) {
```

Comment:
Actually, just realised we should probably refactor these mixins to use the new variables too.